bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/graphics/Device.java

```diff
@@ -33,6 +33,8 @@
  */
 public abstract class Device implements Drawable {
 
+	static boolean strictChecks = Boolean.getBoolean("org.eclipse.swt.internal.enableStrictChecks");
```

Comment:
Should this be `System.getProperty("org.eclipse.swt.internal.enableStrictChecks") != null` for the sake of consistency as well?

Otherwise, `-Dorg.eclipse.swt.internal.enableStrictChecks` will enable the checks on GTK but not on Windows. On Windows, you explicitly have to assign "true". This also means that the test configuration for GitHub Actions and Jenkins are inconsistent in this PR, as GitHub Actions will enable strict mode on Windows, while Jenkins will not.